zetaclient/btc_signer.go

```diff
@@ -93,9 +95,16 @@ func (signer *BTCSigner) SignWithdrawTx(to *btcutil.AddressWitnessPubKeyHash, am
 		return nil, err
 	}
 
+	// size checking
+	txSize := uint64(tx.SerializeSize())
+	if txSize > sizeLimit {
+		fmt.Printf("tx size %d is greater than sizeLimit %d; use sizeLimit", txSize, sizeLimit)
+		txSize = sizeLimit
+	}
```

Comment:
It seems a good workaround currently to ensure we don't pay more fees on the TSS side compared to the fees paid by user.
However, we might want to find another solution like aborting the cctx if the fees are too low because we might end up with a stale transaction.